apps/webClient/src/views/Home/index.tsx

```diff
@@ -32,71 +31,53 @@ const Home: React.FC<Props> = ({}) => {
-            <Container maxWidth="xs">
-              <Grid container justify="center" alignItems="center">
-                <Typography variant="body1">03.</Typography>
-                <Typography
-                  gutterBottom
-                  variant="h4"
-                  className={classes.contrastHeader}
-                >
-                  about us
-                </Typography>
-              </Grid>
-              <Typography variant="body1" align="center" color="textSecondary">
-                <i>
-                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-                  do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-                  Auctor elit sed vulputate mi. Orci eu lobortis elementum nibh
-                  tellus molestie nunc non blandit. Mi in nulla posuere
-                  sollicitudin aliquam ultrices sagittis.
-                </i>
+          <Grid item container sm={4} justify="center">
+            <Grid item container justify="center" alignItems="center">
+              <Typography
+                gutterBottom
+                variant="h4"
+                className={classes.contrastHeader}
+              >
+                oh hello
               </Typography>
-            </Container>
-          </Grid>
-          <Grid item container sm={3} justify="flex-end">
-            <NavButton route="about" text="about us" />
-          </Grid>
-        </Grid>
-      </Box>
-      <Box mt={12}>
-        <Grid container justify="space-around">
-          <Container maxWidth="xs">
-            <Typography
-              variant="h5"
-              align="center"
-              className={classes.contrastHeader}
-              gutterBottom
-            >
-              partner name
-            </Typography>
+            </Grid>
             <Typography variant="body1" align="center" color="textSecondary">
               <i>
                 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-                sitame etel
+                eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                Auctor elit sed vulputate mi. Orci eu lobortis elementum nibh
+                tellus molestie nunc non blandit. Mi in nulla posuere
+                sollicitudin aliquam ultrices sagittis.
               </i>
             </Typography>
-          </Container>
-          <Container maxWidth="xs">
-            <Typography
-              variant="h5"
-              align="center"
-              className={classes.contrastHeader}
-              gutterBottom
-            >
-              partner name
-            </Typography>
-            <Typography variant="body1" align="center" color="textSecondary">
-              <i>
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-                sitame etel
-              </i>
-            </Typography>
-          </Container>
+            <Box mt={2}>
```

Comment:
I think that `Grid containers` should only have `Grid item` for everything to work properly.  Move the Box inside the grid item.